test/docgen/assemblies.ts

```diff
@@ -24,7 +26,7 @@ export class Assemblies {
 
   private constructor() {
     this.ts = new reflect.TypeSystem();
-    this.addAssemblies(`${__dirname}/../__fixtures__/assemblies`);
```

Comment:
It's an invalid closure by definition so I opted towards just adding another set. I could upgrade the existing set of fixtures but wanted to avoid unrelated snapshot updates.